libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/util/HapticFeedback.kt

```diff
@@ -0,0 +1,43 @@
+package com.mapbox.navigation.dropin.util
+
+import android.content.Context
+import android.os.Build
+import android.os.VibrationEffect
+import android.os.Vibrator
+import android.os.VibratorManager
+
+/**
+ * Facade for OS Vibrator.
+ */
+internal class HapticFeedback(context: Context) {
+
+    internal val vibrator = if (Build.VERSION_CODES.S <= Build.VERSION.SDK_INT) {
+        (context.getSystemService(Context.VIBRATOR_MANAGER_SERVICE) as? VibratorManager)
+            ?.defaultVibrator
+    } else {
+        (context.getSystemService(Context.VIBRATOR_SERVICE) as? Vibrator)
+    }
+
+    fun tick() {
+        vibrate(TICK_DURATION_MS)
+    }
```

Comment:
Any reason we need this function? Can we just have 
```
fun vibrate() {
    if (Build.VERSION_CODES.O <= Build.VERSION.SDK_INT) {
        vibrator?.vibrate(
            VibrationEffect.createOneShot(
                 TICK_DURATION_MS,
                 VibrationEffect.DEFAULT_AMPLITUDE
            )
        )
    } else {
        vibrator?.vibrate(milliseconds)
    }
}
```
I think this should be enough because this duration is not something that can be customized in the form of an option.

Reply:
That would decrease usability and limit this class to do only one thing.